### PR TITLE
Updating custom attributes when referral param is passed

### PR DESCRIPTION
**Pull request by cmm-apli**
## Description

This change is required to trigger conversation_updated webhook whenever referral parameter is present on URL and passed as part of SDK input parameters, so we can achieve the "ideal" experience within nane (backend chat bot) integration.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Local tests executed manually:
image

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/packs/sdk.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 } from '../sdk/cookieHelpers';
 import { addClasses, removeClasses } from '../sdk/DOMHelpers';
 import { SDK_SET_BUBBLE_VISIBILITY } from 'shared/constants/sharedFrameEvents';
-const runSDK = ({ baseUrl, websiteToken, referral }) => {
+const runSDK = ({ baseUrl, websiteToken, referral, ref }) => {
```
Comment (Behtsa) [inline app/javascript/packs/sdk.js:15]: Does referral and ref have the same value? wdyt @Arjona99

**Review by AlanDavd — APPROVED** [review]
LGTM! Quede asi 🤯

File: app/javascript/packs/sdk.js (hunk 1 above)
Comment (Arjona99) [reply app/javascript/packs/sdk.js:15]: Yep, `referral` has the same purpose of `ref` variable @cmm-apli , so there is no need to receive a new param

**Review by Arjona99 — CHANGES_REQUESTED** [review]
Just fix the ref variable 👍🏼  And could you add to the PR an evidence of how nane knows that the conversation has been updated please?

File: app/javascript/packs/sdk.js (hunk 1 above)
Comment (cmm-apli) [reply app/javascript/packs/sdk.js]: done

**Review by Arjona99 — APPROVED** [review]
:shipit:

**Review by Arjona99 — APPROVED** [review]

**Pull request merged**